### PR TITLE
Add support for other linux architectures on travis

### PR DESCRIPTION
**Pull request by djhoese**
Similar to #434, but for ppc64le which is distributed along side the x86_64 install scripts by anaconda.

### CODE REVIEW

**Comment by djhoese** [comment]
I didn't add support for arm64 because I couldn't find a recent supported build on the anaconda download site: https://repo.anaconda.com/miniconda/

**Comment by bsipocz** [comment]
@djhoese - we cannot run tests on travis any more (they pulled the support for open source). So if you prefer, I can merge this as is now. 

Otherwise, my suggestion would be to start moving over to other CI providers.

**Comment by djhoese** [comment]
>  we cannot run tests on travis any more (they pulled the support for open source)

? Announcment?

**Comment by bsipocz** [comment]
Longer description is in this astropy-dev mail: https://groups.google.com/g/astropy-dev/c/45frnTEAX-U

Their announcement: https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

**Comment by djhoese** [comment]
Well great, I never got any email about this.

**Comment by bsipocz** [comment]
It's all quite quick. Also, if you haven't got migrated off from travis.org, you still have some time I believe, sadly we did that migration, and run into this sudden surprise a few weeks later.

**Comment by pllim** [comment]
@djhoese , Actions has been added to `package-template` and there is an open PR at astropy/astropy#11045 for different archs. Sorry for any inconvenience caused!

**Comment by bsipocz** [comment]
@pllim - I would have actually preferred to get this merged. If this helps @djhoese while they still have the Travis credit, I'm happy for it, even if ci-helpers is no use for astropy any more

**Comment by pllim** [comment]
Ops, sorry. I did not know that plan. I re-opened the PR but you will have to probably test your patch somewhere else to make sure it really works.

**Comment by bsipocz** [comment]
@djhoese - could you please let us know whether you have tested the solution in this branch?

**Comment by djhoese** [comment]
I was only adding this here because of a contribution to one of my own projects (https://github.com/pytroll/pydecorate/pull/16) where the contributor wanted ppc64le added. In their PR they had a separate execution block in the CI config just to handle the pcc case because ci-helpers didn't allow for it. This PR was my attempt to simplify that. I'm ok closing this as we will probably be switching to github actions for our main testing. I definitely don't want to waste any travis minutes we get on extra platform testing unless it is absolutely necessary.

Bottom line: Feel free to close.

**Comment by bsipocz** [comment]
I'm happy to merge this without being explicitly tested, if you're happy with the small risk that it may break your CI (I can't spot any issues in the review).

**Comment by djhoese** [comment]
Let's close it. I'll leave the branch where it is. And if things change with travis and how my projects use it then maybe we'll revisit this and I can more thoroughly test it.

**Comment by mwcraig** [comment]
@djhoese -- let us know if he need any help migrating to actions. Your contributions here have been very much appreciated.

**Comment by djhoese** [comment]
We've started migrating some projects. With the conda incubator it doesn't seem too bad. Just not something I was planning on dealing with this close to so many deadlines. Thanks for the offer.

**Comment by mwcraig** [comment]
> Just not something I was planning on dealing with this close to so many deadlines. 

This ×💯
